docs/contributing/CHANGELOG_TEMPLATE.md

```diff
@@ -2,28 +2,23 @@
 
 ## Unreleased
 
-💥 Breaking changes:
+Breaking changes
```

Comment:
@EoinShaughnessy Sorry I should have spotted this in my first review - should we make these h3's? E.g:

```suggestion
### Breaking changes
```

Reply:
D'oh my bad, yes you're right, ta for flagging! Will change here and anywhere else affected.